test/library/standard/Heap/isEmpty/heapIsEmpty.chpl

```diff
@@ -0,0 +1,7 @@
+use Heap;
+var h = new heap(int);
+writeln(h.isEmpty());
+h.push(1);
+writeln(h.isEmpty());
+h.pop();
+writeln(h.isEmpty());
```

Comment:
It'd be nice to add a newline to the last line.  github indicates its absence by that red circle-dash icon.

Reply:
Heh, it doesn't indicate it here in the PR overview page excerpt for my comment, just in the actual "Files changed" view.